lib/rubocop/cop/style/symbol_proc.rb

```diff
@@ -40,6 +40,7 @@ def on_block(node)
             return if proc_node?(send_or_super)
             return if %i[lambda proc].include?(block_method_name)
             return if ignored_method?(block_method_name)
+            return if block_arg.source.include?(',')
```

Comment:
What if one of the args is "dfdome, dfdf"? Doesn't seem like a very reliable check to me.

Reply:
Thanks for the review.
When multiple block arguments are given, I thought that it works well because this cop does not register offenses. However, it was not good that the condition was ambiguous.

I changed this condition as follows.

```diff
diff --git a/lib/rubocop/cop/style/symbol_proc.rb b/lib/rubocop/cop/style/symbol_proc.rb
index 243e160f4..4bc3c62b0 100644
--- a/lib/rubocop/cop/style/symbol_proc.rb
+++ b/lib/rubocop/cop/style/symbol_proc.rb
@@ -31,7 +31,7 @@ module RuboCop
         end

         def on_block(node)
-          symbol_proc?(node) do |send_or_super, block_arg, method|
+          symbol_proc?(node) do |send_or_super, block_args, method|
             block_method_name = resolve_block_method_name(send_or_super)

             # TODO: Rails-specific handling that we should probably make
@@ -40,7 +40,8 @@ module RuboCop
             return if proc_node?(send_or_super)
             return if %i[lambda proc].include?(block_method_name)
             return if ignored_method?(block_method_name)
-            return if block_arg.source.include?(',')
+            return if block_args.children.size == 1 &&
+                      block_args.source.include?(',')

             offense(node, method, block_method_name)
           end
```

Is there a case to worry about?